enhancements/network/no_overlay_mode.md

```diff
@@ -0,0 +1,807 @@
+
+### Non-Goals
+See the upstream OVN-Kubernetes enhancement for detailed non-goals:
+https://github.com/ovn-kubernetes/ovn-kubernetes/blob/master/docs/okeps/okep-5259-no-overlay.md#non-goals
+
+## Proposal
+
+<!-- This section should explain what the proposal actually is. Enumerate -->
+<!-- *all* of the proposed changes at a *high level*, including all of the -->
+<!-- components that need to be modified and how they will be -->
+<!-- different. Include the reason for each choice in the design and -->
+<!-- implementation that is proposed here. -->
+
+<!-- To keep this section succinct, document the details like API field -->
+<!-- changes, new images, and other implementation details in the -->
+<!-- **Implementation Details** section and record the reasons for not -->
+<!-- choosing alternatives in the **Alternatives** section at the end of -->
+<!-- the document. -->
+
+The no-overlay feature largely leverages the existing BGP functionality in OVN-Kubernetes and only needs few API changes.
+The feature can be applied to:
+- the default network, at cluster installation time
+- Cluster User Defined Networks (CUDNs), at run time
+
+For each network we are going to need a `transport` parameter that takes `geneve` (default) or `no-overlay`.
+Then if `transport` is set to `no-overlay`, we need the following parameters to configure the no-overlay mode:
+- `outboundSNAT`:
+  - `Enabled`: apply source NAT to egress traffic, allowing only the node IP to be exposed, which is today's expected behaviour unless EgressIP is used;
+  - `Disabled`: do not apply any SNAT to egress traffic, thus exposing the pod subnet outside the cluster.
+- `routing`:
+  - `Managed`: delegate to OCP the configuration of the BGP routers or reflectors necessary to advertise the network pod subnet to all cluster nodes;
+  - `Unmanaged`: use the FRRConfig and RouteAdvertisements provided by the cluster administrator to implement the no-overlay mode
+
+For CUDNs these parameters are to be added to the CUDN CRD and can be configured by the cluster administrator when creating a CUDN. For the default network, these parameters must be input by the cluster administrator at installation time and passed over to ovn-kubernetes by the Cluster Network Operator.
+
+**MTU Considerations:** Networks operating in no-overlay mode use the provider network MTU directly, as there is no encapsulation overhead. This means pods on no-overlay networks can use the full MTU of the underlying physical network infrastructure, unlike overlay networks which require a reduced MTU to accommodate encapsulation headers.
+
+There are also two global parameters specific to the way that the no-overlay mode is to be implemented when `NoOverlayOptions.routing`=`Managed`, affecting the generated BGP configuration:
+- `asNumber`: (optional) the Autonomous System (AS) number to be used in the generated FRRConfig. When omitted, this defaults to 64512.
```

Comment:
We shall state that this asNumber is for the default VRF. And it shall be aligned with other FRRconfigurations added by the cluster admin.